functions.php

```diff
@@ -9,27 +9,28 @@
-	'/template-tags.php',                   // Custom template tags for this theme.
-	'/pagination.php',                      // Custom pagination for this theme.
-	'/hooks.php',                           // Custom hooks.
-	'/extras.php',                          // Custom functions that act independently of the theme templates.
-	'/customizer.php',                      // Customizer additions.
-	'/custom-comments.php',                 // Custom Comments file.
-	'/jetpack.php',                         // Load Jetpack compatibility file.
-	'/wp-custom-fields.php',                // Disable WP Core custom fields metaboxes.
-	'/tgm-plugin-activation.php',           // Load TGM Plugin Activation script for dependent plugin recommendations.
-	'/class-wp-social-media-walker.php',    // Load custom WordPress nav walker for footer Social Media menu wdiget.
-	'/wp-custom-menu.php',                  // Load custom menu builder functions.
-	'/editor.php',                          // Load Editor functions.
-	'/deprecated.php',                      // Load deprecated functions.
+	'/theme-sidebars-helpers.php',               // Sidebar helper functions.
+	'/theme-settings.php',                       // Initialize theme default settings.
+	'/setup.php',                                // Theme setup and custom theme supports.
+	'/theme-activation.php',                     // Build sample menus upon theme activation.
+	'/asu-favicons.php',                         // Enable ASU Favicons.
+	'/wpautop.php',                              // Disable wpautop.
+	'/widgets.php',                              // Register widget area.
+	'/enqueue.php',                              // Enqueue scripts and styles.
+	'/template-tags.php',                        // Custom template tags for this theme.
+	'/pagination.php',                           // Custom pagination for this theme.
+	'/hooks.php',                                // Custom hooks.
+	'/extras.php',                               // Custom functions that act independently of the theme templates.
+	'/customizer.php',                           // Customizer additions.
+	'/custom-comments.php',                      // Custom Comments file.
+	'/jetpack.php',                              // Load Jetpack compatibility file.
+	'/wp-custom-fields.php',                     // Disable WP Core custom fields metaboxes.
+	'/tgm-plugin-activation.php',                // Load TGM Plugin Activation script for dependent plugin recommendations.
+	'/class-wp-social-media-walker.php',         // Load custom WordPress nav walker for footer Social Media menu wdiget.
+	'/wp-custom-menu.php',                       // Load custom menu builder functions.
+	'/editor.php',                               // Load Editor functions.
+	'/deprecated.php',                           // Load deprecated functions.
+	'/class-uds-notification-banner.php', // Load the UDS notification widget.
```

Comment:
There isn't a hard rule about the order of these includes, but there is a little logic. Early bootstrap logic when the theme is initializing and setup is early i this list and the load order should somewhat make sense in terms of dependencies or overrides.

Your new include file, "class-uds-notification-banner.php" is setting up a new widget area, it should probably load following the widgets.php include.

Reply:
Pushed a new commit to move that include directly underneath 'widgets.php'